saws/completer.py

```diff
@@ -137,10 +137,20 @@ def replace_substitution(self, text):
             A string representing input command text with a substitution,
             if one has been found.
         """
-        if '%s' in text:
+        text_to_find = '%s'
+
+        if text_to_find in text:
             tokens = text.split()
-            text = ' '.join(tokens[:-1])
-            text = re.sub('%s', tokens[-1], text)
+            text_to_find_index = self.text_utils.get_token_index(text_to_find,
+                                                                 tokens)
+            try:
```

Comment:
this try is also necessary to avoid errors on autocomplete when user tries to autocomplete on

```
saws> aws ec2 ls --ec2-tag-key
```

also if the user doesn't complete the command instead of failing (with list index not found) it will just return 

```
saws> aws ec2 ls --ec2-tag-key
{
    "Reservations": []
}
```

which I believe to be better output